include/rocksdb/perf_flags.h

```diff
@@ -0,0 +1,104 @@
+#pragma once
+#include <stdint.h>
+
+namespace rocksdb {
+
+struct PerfFlags {
+  uint8_t perf_level;
+
+  uint8_t enable_user_key_comparison_count_bit : 1;          // 2
```

Comment:
Your arrangement of fields only introduce the difficulty of adding new fields, while bringing no advantages? For example, in our case we even won't use these levels. Set level is not called frequently as well. So that there are no performance benefits. Only maintenance costs.